tools/twix/src/panels/map/layers/behavior_simulator.rs

```diff
@@ -38,6 +51,21 @@ impl Layer for BehaviorSimulator {
                 width: 0.02,
                 color: Color32::BLACK,
             };
+
+            let Ok(motion_command): Result<MotionCommand> = self.motion_command.0[player_number].parse_latest() else {
+                continue
+            };
+
+            if let MotionCommand::Walk { path, .. } = motion_command {
```

Comment:
The let-else-continue here makes it harder to add more stuff later.
What do you think of this?
```suggestion
            if let Ok(MotionCommand::Walk { path, .. }) =
                self.motion_command.0[player_number].parse_latest()
```

Reply:
done